pkg/opensearch/response_parser.go

```diff
@@ -1012,7 +1016,8 @@ func (rp *responseParser) processAggregationDocs(esAgg *simplejson.Json, aggDef
 		for _, e := range fields {
 			for _, propKey := range propKeys {
 				if e.Name == propKey {
-					e.Append(props[propKey])
+					value := props[propKey]
+					e.Append(&value)
```

Comment:
Not even a nit, just fyi the `utils.Pointer` method (which exists in a number of different repos) can be nice in places like this:
```suggestion
					e.Append(utils.Pointer(props[propKey]))
```